src/interfaces/node.cpp

```diff
@@ -257,6 +260,12 @@ class NodeImpl : public Node
     {
         return MakeWallet(LoadWallet(*m_interfaces.chain, name, error, warning));
     }
+    std::unique_ptr<Wallet> createWallet(const std::string& name, std::string& error, std::string& warning, const SecureString& passphrase, uint64_t wallet_creation_flags) override
+    {
+        std::shared_ptr<CWallet> wallet;
+        WalletCreationStatus status = CreateWallet(*m_interfaces.chain, passphrase, wallet_creation_flags, name, error, warning, wallet);
```

Comment:
Unused variable `status`. Also it would be more consistent to make the interface method match the new pattern.

Reply:
Changed the interface to match the new pattern.